cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -1041,9 +1041,9 @@ private predicate flowCandFwd0(Node node, boolean fromArg, AccessPathFront apf,
   )
   or
   exists(Content f, AccessPathFront apf0 |
-    flowCandFwdRead(f, node, apf0, fromArg, config) and
+    flowCandFwdRead(f, node, fromArg, apf0, config) and
     apf0.headUsesContent(f) and
```

Comment:
I think this line ought to be pushed into `flowCandFwdRead` as well.

Reply:
(and then the `apf` argument can be projected away)

Reply:
Makes sense.